services/requirements.txt

```diff
@@ -0,0 +1,18 @@
+beautifulsoup4==4.9.1
+certifi==2020.6.20
+chardet==3.0.4
+idna==2.8
+jellyfish==0.5.6
+lxml==4.5.2
+marshmallow==3.0.0rc1
+meson==0.55.0
+numpy==1.19.1
+pandas==1.1.1
+python-dateutil==2.8.1
+pytz==2020.1
+PyYAML==3.13
+requests==2.22.0
+six==1.15.0
+soupsieve==2.0.1
+urllib3==1.25.10
+us==1.0.0
```

Comment:
We should combine the requirements.txt into a single file. Also I would test out the functions once you do that so we know it integrates properly

Reply:
You will call `scrape_indeed()` in the main.py file